rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/terminal/ITerminalIDEServer.java

```diff
@@ -482,9 +484,30 @@ public String getMainModule() {
 
         @Override
         public String toString() {
-            return "LanguageParameter(pathConfig=" + pathConfig + ", name=" + name + ", extension=" + extension
+            return "LanguageParameter(pathConfig=" + pathConfig + ", name=" + name + ", extensions=" + extensions
```

Comment:
The `toString` of an array is probably not what you want here

Reply:
tnx, fixed.